en/docs/administer/configure-approvals-for-choreo-workflows.md

```diff
@@ -16,16 +16,18 @@ Click the respective tab for details on permissions depending on the workflow fo
 
      To review and respond to environment promotion approval requests, a user must have the following permissions. Administrators must ensure that users designated to review and respond to approval requests have these permissions:
 
-      - **WORKFLOW-MANAGEMENT**: Grants access to view and approve workflow requests. Each workflow type has a separate permission.
+      - **WORKFLOW-MANAGEMENT**: 
+          - Approve component promotion requests: Grants access to review and approve environment promotion requests.
```

Comment:
Since we are going with "Approve component promotion requests:" shall we have the description as "Grants access to review and approve the promotion of components to critical environments."

Reply:
Alrighty